firedantic/_async/model.py

```diff
@@ -62,6 +86,7 @@ class AsyncBareModel(pydantic.BaseModel, ABC):
     __collection__: Optional[str] = None
     __document_id__: str
     __ttl_field__: Optional[str] = None
+    __composite_indexes__: Optional[List[IndexDef]]
```

Comment:
Random thought of the day. We convert this to a set, so why not allow this to be a set, list or even a tuple?